content/chat/reactions.textile

```diff
@@ -5,3 +5,138 @@ product: chat
 languages:
   - javascript
 ---
+
+In most chat environments you will have encountered some form of message reactions. The ability to add a reaction such as an emoji to a fellow chat participant's message is an important piece of functionality for users to express their opinion without replying with text.
+
+<img src="/images/products/chat/reactions.png" alt="Example of a chat message sent by a Summer Ouellet, with multiple reactions below.">
+
+h2(#enable). Enable message reactions
```

Comment:
I think anything I mentioned in replies and threads is likely relevant from this section onwards 🙂

Reply:
https://github.com/ably/docs/pull/2090/commits/f6dd39f2a60d18b0bb264bc125d9bdc1d46b422e

Yup! various parts of reactions updated accordingly.